modules/opensearch/src/OpenSearchTingObject.php

```diff
@@ -200,7 +213,19 @@ public function getSubjects() {
    * {@inheritdoc}
    */
   public function getLanguage() {
-    return $this->firstEntry($this->getRecordEntry('dc:language'));
+    if (!isset($this->reply->record['dc:language'])) {
```

Comment:
Shouldn't this be `$this->firstEntry($this->getRecordEntry)`?

Reply:
`$this->firstEntry($this->getRecordEntry('dc:language'))` will return plain string, but in case there are multiple languages we need an array for processing and returning the list of languages.

Reply:
> Are you sure about this? I'm quite sure that if there is multiple languages the language code is just "mul" instead of "da", "en", etc.

No. In case when language code is "mul", the "mul" is the first element, next are listed other languages available.
Example: https://oss-services.dbc.dk/opensearch/5.0/?action=search&query=%22Kvartalsskriftet%22&agency=100200&profile=test&start=1&stepValue=5

> If its an array, why is the variable name in L220 called $languages_string?

Because array values with language name are concatenated in one string I suppose.

Reply:
Ok, should it be `$this->getRecordEntry` instead then? I dont see `$this->reply` used anywhere else in this file.

Reply:
In case of `$this->getRecordEntry` will be returned **"Flere sprog"** string and rest of items will be skipped.